connect-inject/common_inject_configuration.go

```diff
@@ -0,0 +1,160 @@
+package connectinject
+
```

Comment:
this file could potentially read `metrics_configuration.go`?

Reply:
agree on the naming bits here!